RuntimeUnityEditor/Windows/ObjectTree/ObjectTreeViewer.cs

```diff
@@ -102,27 +103,29 @@ private void DisplayObjectTreeHelper(GameObject go, int indent, ref int currentC
         {
             currentCount++;
 
-            var needsHeightMeasure = _singleObjectTreeItemHeight == 0;
+            var needsHeightMeasure = _singleObjectTreeItemHeight == 0f;
```

Comment:
Initially thought the int was rounding the _singleObjectTreeItemHeight, turns out the entire issue here is that GetLastRect doesn't include the margin, so each object should be 22 units instead of 18 (margin of 4). The first entry has margin top and bottom, all subsequent entries are just bottom (they overlap neighbors), so the math you'll see below everywhere for total current size is _singleObjectTreeItemHeight (which is not 22, not 18) + margin (to account for the first items top margin). Left it as a float since it should be and makes it future proof (UI scaling etc?)